### DISCUSSION TITLE
When can support for Ubuntu 23.10 be expected?

### DISCUSSION THREAD

**Discussion by DEPSTRCZ** [RaspAP/raspap-webgui#1487] [General]
I have installed the latest version of Ubuntu server. And now i have realized that it is not supported. So i a mthinking if i should reinstall my os to supported one. Or just wait.

**Comment by billz** [comment]
New Ubuntu Server releases generally occur every six months. This cadence isn't practical for us to support, so we will likely switch to focusing on LTS releases only. That said, RaspAP will _probably_ run fine on 23.10.

**Comment by billz** [comment]
Alternatively, you can now also deploy RaspAP in a Docker container.
https://docs.raspap.com/docker/
